src/main/java/com/mixpanel/android/mpmetrics/MPConfig.java

```diff
@@ -208,7 +212,16 @@ public int getBulkUploadLimit() {
 
     // Target max milliseconds between flushes. This is advisory.
     public int getFlushInterval() {
-        return mFlushInterval;
+        return getFlushInterval(null);
```

Comment:
Hmm yeah that's true in this case but IMO it's best to have a single source of truth